CatBreeds/Model/CatImageHandler.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  CatImageHandler.swift
+//  CatBreeds
+//
+//  Created by Areeb Durrani on 4/10/25.
+//
+
+import Foundation
+
+struct Thumbnail: Decodable {
+    let source: String
+}
+
+struct Page: Decodable {
+    let thumbnail: Thumbnail
+}
+
+struct Query: Decodable {
+    let pages: [Page]
+}
+
+struct ImageResponse: Decodable {
+    let query: Query
+}
+
+func removePunctuation(from string: String, delimiter: Character) -> String {
+    var s = string
+    if s.contains(delimiter) {
+        s.removeSubrange(s.firstIndex(of: delimiter)!..<s.endIndex)
+    }
+    return s
```

Comment:
If this func removes any given character, rename it in that way, not name as removePunctuation

Reply:
Better write this func as an extension of String